### PR TITLE
Add papers using Larq to FAQ; closes #13

### LINKED ISSUE #13: Bring back "papers using larq"
[Papers using larq](https://github.com/larq/docs/blob/master/docs/papers.md) should be added to FAQ again, currently it is not linked in the website

### PR DESCRIPTION
**Pull request by leonoverweel**

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -32,7 +32,22 @@ If Larq helps you in your work or research, it would be great if you can cite it
...
 ```
 
-If your paper is publicly available, feel free to also add it to the list of [Papers using Larq](https://github.com/larq/docs/edit/master/docs/papers.md).
+If your paper is publicly available, feel free to also add it to the list of papers using Larq below.
+
+## What papers have used Larq?
+
+One of the focuses of Larq is to accelerate research on neural networks with extremely low precision weights and activations.
+Here is a list of papers that have used Larq:
+* Helwegen, Koen, et al. ["Latent weights do not exist: Rethinking binarized neural network optimization."](https://papers.nips.cc/paper/8971-latent-weights-do-not-exist-rethinking-binarized-neural-network-optimization.pdf) *Advances in Neural Information Processing Systems (NeurIPS)*. 2019.
```
Comment (lgeiger) [inline docs/faq.md]: This doesn't really render well on the website: https://docs-ezf5lam29.now.sh/faq/#what-papers-have-used-larq

I think you might need to add some empty lines.

Comment (leonoverweel) [reply docs/faq.md]: Added a blank line in dc1739d.

**Comment by leonoverweel** [comment]
When building locally following the contributing instructions, I ran into:

```
MkDocs encountered as error parsing the configuration file: while constructing a Python object
cannot find module 'plot_altair' (No module named 'plot_altair')
  in "/Users/leon/Repositories/Plumerai/docs/mkdocs.yml", line 77, column 19
```

Is that an error on my end?

**Comment by lgeiger** [comment]
> Is that an error on my end?

@koenhelwegen Had similar errors inside his venv

File: docs/faq.md [hunk 2]
```diff
@@ -32,7 +32,23 @@ If Larq helps you in your work or research, it would be great if you can cite it
...
-If your paper is publicly available, feel free to also add it to the list of [Papers using Larq](https://github.com/larq/docs/edit/master/docs/papers.md).
+If your paper is publicly available, feel free to also add it to the list of papers using Larq below.
+
+## What papers have used Larq?
+
+One of the focuses of Larq is to accelerate research on neural networks with extremely low precision weights and activations.
+Here is a list of papers that have used Larq:
+
+* Helwegen, Koen, et al. ["Latent weights do not exist: Rethinking binarized neural network optimization."](https://papers.nips.cc/paper/8971-latent-weights-do-not-exist-rethinking-binarized-neural-network-optimization.pdf) *Advances in Neural Information Processing Systems (NeurIPS)*. 2019.
+  * Optimization of Binarized Neural Networks (BNNs) currently relies on real-valued latent weights to accumulate small update steps.
```
Comment (lgeiger) [inline docs/faq.md]: I think you need two more indents here: https://docs-q69010jxj.now.sh/faq/#what-papers-have-used-larq

Comment (leonoverweel) [reply docs/faq.md]: Seems it works now, thanks.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -16,6 +16,8 @@ python generate_api_docs.py
 mkdocs serve
```
Comment (lgeiger) [inline CONTRIBUTING.md]: Let's always recommend it :+1:

```suggestion
python -m mkdocs serve
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -16,6 +16,8 @@ python generate_api_docs.py
...
 ```
 
+If `mkdocs serve` doesn't work, try `python3 -m mkdocs serve`.
+
```
Comment (lgeiger) [inline CONTRIBUTING.md]: See above
```suggestion

```

**Pull request merged**